views/update/update-presenter.html

```diff
@@ -1,6 +1,6 @@
 {% extends "update-layout.html" %}
 
-{% set title = "Who can we contact about this update?" %}
+{% set title = "Who can we contact about this application?" %}
```

Comment:
Not sure we should be changing the text here for the existing Update journey. We discussed using the example code in `update-layout.html` which would allow us to use the 'journey' param to vary this...

So something like:

{% if (journey == "remove") %}
  {% set title = "Who can we contact about this application?" %}
{% else %}
  {% set title = "Who can we contact about this update?" %}
{% endif %}

Did that work?

Reply:
Think that this is fine now. @dlloyd1-ch ?

Reply:
issue fixed